hypha/apply/funds/models/applications.py

```diff
@@ -343,9 +344,73 @@ def get_form_parameters(self, submission_id=None):
 
         return form_parameters
 
+    def get_form_fields(self, draft=False, posted_field_values={}):
```

Comment:
I didn't mean to write all the get_form_fields code here, instead I was thinking if we could override it, like I have done [here](https://github.com/OpenTechFund/hypha/blob/e7c64e0223f4db1fcfd62fbc3ebe0f88fbc25584/hypha/apply/funds/models/applications.py#L346) for this [pr](https://github.com/OpenTechFund/hypha/pull/2162).

But anyway, I think for this issue it would be good to do it the way you were doing it earlier. As we would also need this in other places where we use group fields. E.g. to get this working for submission edit form, you would also need to send form_data in the [edit form](https://github.com/OpenTechFund/hypha/blob/f6a1934de94a4a349be62ea55706aa5f0ba463cc/hypha/apply/funds/views.py#L945). I think you can use `self.object.raw_data`.

So can you revert your changes to do this in [get_form_fields](https://github.com/OpenTechFund/hypha/blob/f6a1934de94a4a349be62ea55706aa5f0ba463cc/hypha/apply/stream_forms/models.py#L49) only.

Reply:
Changes are reverted